JSDemos/Demos/ColorBox/Overview/description.md

```diff
@@ -1 +1,31 @@
-The ColorBox is&nbsp;a&nbsp;UI component that allows an&nbsp;end user to&nbsp;enter a&nbsp;color or&nbsp;pick it&nbsp;out from the drop-down editor.
\ No newline at end of file
+The ColorBox allows users to enter a color or pick it out from the drop-down editor. When you create a ColorBox, specify its [value]() property to set an initial color. Users can change the value instantly after they picked a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property. 
+
+You can hide the editor's drop-down button to allow users to only type in a color code. To do this, disable the [showDropDownButton]() property. Users can also edit the transparency of the picked shade if the [editAlphaChannel]() property is set to **true**.
+
+## Configure the ColorBox
+
+Use the following properties to control the component interactivity:
```

Comment:
```suggestion
Use the following properties to control the component's interactivity:
```

Do you mean 'to specify the component's interactivity'?

Reply:
Yes, I mean that